dotcom-rendering/src/model/enhance-H3s.ts

```diff
@@ -0,0 +1,92 @@
+import { JSDOM } from 'jsdom';
+
+const isFalseH3 = (element: CAPIElement): boolean => {
+	if (!element) return false;
+	// Checks if this element is a 'false h3' based on the convention: <p><strong><H3 text</strong></p>
+	if (
+		element._type !== 'model.dotcomrendering.pageElements.TextBlockElement'
+	) {
+		return false;
+	}
+	const frag = JSDOM.fragment(element.html);
+	if (!frag || !frag.firstElementChild) return false;
+	const html = frag.firstElementChild.outerHTML;
+	// The following things must be true for an element to be a faux H3
+	const hasPwrapper = frag.firstElementChild.nodeName === 'P';
+	const containsStrongtags =
+		frag.firstElementChild.outerHTML.includes('<strong>');
+	const doesNotContainLinks =
+		!frag.firstElementChild.outerHTML.includes('<a>');
+	const startStrong = html.startsWith('<p><strong>');
+	const endsStrong = html.endsWith('</strong></p>');
+
+	return (
+		hasPwrapper &&
+		containsStrongtags &&
+		doesNotContainLinks &&
+		startStrong &&
+		endsStrong
+	);
+};
+
+const extractH3 = (element: CAPIElement): string => {
+	// Extract the text based on the convention: <p><strong><H3 text</strong></p>
+	const textElement = element as TextBlockElement;
+	const frag = JSDOM.fragment(textElement.html);
+	if (isFalseH3(element)) {
+		return (
+			frag.firstElementChild?.innerHTML
+				.split('<strong>')
+				.join('')
+				.split('</strong>')
+				.join('') ?? ''
```

Comment:
It might be more explicit to use `replaceAll` here, if what we’re doing is stripping away the `strong` tags.

```suggestion
			frag.firstElementChild?.innerHTML
				.replaceAll('<strong>', '')
				.replaceAll('</strong>', '') ?? ''
```